src/App.tsx

```diff
@@ -15,47 +15,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
+  goods: string[], sortType : SortType, isReversed : boolean,
```

Comment:
Keep your code style consistent.
```suggestion
  goods: string[], sortType: SortType, isReversed: boolean,
```